### DISCUSSION TITLE
About numerical methods

### DISCUSSION THREAD

**Discussion by zyx1xyz** [AMReX-Combustion/PeleC#859] [Q&A]
Hi, everyone
I would like to confirm my understanding of these options and would appreciate it if you could point out my misunderstanding.

**Time**
do_mol = 1                # Use the MOL method

- mol_iters = 1     # First-order accuracy?
- mol_iters = 2     # Second-order accuracy?

do_mol = 0                # Use the SDC method

- sdc_iters = 1      # First-order accuracy?
- sdc_iters = 2      # Second-order accuracy?

**Convection**
The text mentions two PPM methods:
1.	PLM:
ppm_type = 0
plm_iorder = 4

2.	PPM + WENO:
ppm_type = 1
use_hybrid_weno = true
weno_scheme = 1

The text mentions two mol methods:
mol_iorder = 1 or 2

But there doesn't seem to be any flag that determines whether I chose PPM or mol. 
Is the mol method exclusively used for EB? Additionally, without using EB, is PPM the only method used by default?

**CVODE**
I noticed two types of cvode: GMRES and denseAJ_direct,  which is more suitable for the detonation calculation？

**Comment by SRkumar97** [comment]
Hi, I have the exact same question reg. PPM and MOL methods. The PeleC documentation explains that ppm_type=0 is default. So does this mean that if I do not set any flags reg. numerical scheme, default ppm type 0 will be used? Can someone please explain? 
Thanks in advance!
